<launch>
  <arg name="map_file" default="$(find utexas_gdc)/maps/3ne.yaml" />
  <arg name="door_file" default="$(find clingo_interface)/config/doors.yaml" />
  <arg name="location_file" default="$(find clingo_interface)/config/locations.yaml" />
  <node name="clingo_interface_gui" type="clingo_interface_gui" pkg="clingo_interface_gui">
    <param name="map_file" value="$(arg map_file)" />
    <param name="door_file" value="$(arg door_file)" />
    <param name="location_file" value="$(arg location_file)" />
    <param name="values_file" value="$(find clingo_interface_gui)/data/icapsExp/values" />
    <param name="lua_file" value="$(find clingo_interface_gui)/icapsExp/distances.lua" />
    <rosparam param="obstacle_urdf" command="load" 
      file="$(find clingo_interface)/urdf/obstacle.urdf" />
    <rosparam param="door_urdf" command="load" 
      file="$(find clingo_interface)/urdf/door.urdf" />
  </node>
</launch>
